index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- ABOT HEADER -->
+    <header class="nav-bar">
+        <nav class="nav-section">
+            <p class="logo">My Logo</p>
```

Comment:
- [ ] Kindly wrap the logo with `a` tag, this is for convention, semantics, and user experience(UX) reasons. 👍 Kindy bring the logo out of the `nav` container, a `logo` is not supposed to be part of `nav` items. 👍